CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # minimal-cipher ChangeLog
 
+### 1.4.1 - TBD
+### Changed
+- JSDOC comments in `Cipher.js`.
+- Upgraded eslint to ^7.0.0.
+- Upgraded eslint-plugin-jsdoc to ^ 37.0.0.
+
+### Fixed
+- decrypt helper method to be able to handle multiple chunks.
```

Comment:
All the changes here are related to dev/tests correct?  Which is to say it does not need a release?

I'm not sure about what the fix for a a decrypt helper is, that's in the test suite correct?

Reply:
this is a test suite only fix. There were comments added to the main lib and upgrades to eslint, but no new functionality so need for a release here: https://github.com/digitalbazaar/minimal-cipher/pull/29/commits/abdfe747b26008f6c856b7087307aad3dfd5d492

Reply:
Great, TY.